Orange/statistics/util.py

```diff
@@ -231,3 +231,70 @@ def weighted_mean():
             np.zeros(X.shape[1]),
             nans,
             X.shape[0] - nans))
+
+
+def _sparse_has_zeros(x):
+    """ Check if sparse matrix contains any implicit zeros. """
+    return np.prod(x.shape) != x.data.size
```

Comment:
It is probably better to use `x.nnz` instead of `x.data.size` everywhere.
Looks like the `spmatrix` base class has `nnz` so every type should have it, while e.g. `dok_matrix` does not have `.data`